src/Codeception/Module/REST.php

```diff
@@ -333,6 +334,37 @@ public function amNTLMAuthenticated($username, $password)
     }
 
     /**
+     * Allows to send REST request using AWS Authorization
+     * Requires client to by Guzzle
+     * Example
+     * Config -
+     *
+     * modules:
+     *      enabled:
+     *          - REST:
+     *              aws:
+     *                  key: accessKey
+     *                  secret: accessSecret
+     *                  service: awsService
+     *                  region: awsRegion
+     *
+     * ```php
+     * <?php
+     * $I->amAWSAuthenticated();
+     * ?>
+     * ```
+     * @param array $additionalAWSConfig
+     * @throws ModuleException
+     */
+    public function amAWSAuthenticated($additionalAWSConfig = [])
+    {
+        if (method_exists($this->client, 'setAwsAuth')) {
```

Comment:
I'm not sure what is the best behaviour when client has no `setAwsAuth` method.
One way is to throw an exception, but doing nothing is probably good as well.